src/bid/bid.rs

```diff
@@ -15,44 +14,26 @@ use rand_core::{CryptoRng, RngCore};
 
 #[derive(Copy, Clone, Debug, Default)]
 pub struct Bid {
-    // B^R
-    pub(crate) bid_tree_root: BlsScalar,
-    // sigma^s
-    pub(crate) consensus_round_seed: BlsScalar,
-    // k^t
-    pub(crate) latest_consensus_round: BlsScalar,
-    // k^s
-    pub(crate) latest_consensus_step: BlsScalar,
-    // t_a
-    pub(crate) elegibility_ts: u32,
-    // t_e
-    pub(crate) expiration_ts: u32,
-    // i (One time identity of the prover)
-    pub(crate) prover_id: BlsScalar,
-    // q (Score of the bid)
-    pub(crate) score: Score,
-    // Encrypted value and blinding factor with the bidder key
-    pub(crate) encrypted_data: PoseidonCipher,
-    pub(crate) nonce: BlsScalar,
-
+    // b_enc (encrypted value and blinder)
+    pub encrypted_data: PoseidonCipher,
+    pub nonce: BlsScalar,
     // R = r * G
-    pub(crate) randomness: AffinePoint,
-    // k
-    pub(crate) secret_k: BlsScalar,
+    pub randomness: AffinePoint,
     // m
-    pub(crate) hashed_secret: BlsScalar,
-    // pk (Public Key - Stealth Address)
-    pub(crate) pk: AffinePoint,
+    pub hashed_secret: BlsScalar,
+    // pk (Public Key - Stealth Address) // XXX: Likely to become pk_r
+    pub pk: AffinePoint,
     // c (Pedersen Commitment)
-    pub(crate) c: AffinePoint,
+    pub c: AffinePoint,
 }
 
 impl Bid {
     pub fn init<R>(
```

Comment:
This method name isn't very obvious. It is behaving as a constructor, and `init` implies we already have an instantiated structure and are setting the internal variables

Reply:
Makes sense. Should we call it `new` again?